docs/adr/0005-support-most-recent-action-images.md

```diff
@@ -0,0 +1,29 @@
+# 5. Support of most recent action images (inc Python v2)
+
+Date: 2024-06-07
+
+## Status
+
+Accepted
+
+## Context
+
+We want to provide a Codespaces environment that's intuitive for researchers to understand and that provides a "habitable" environment for them in which to write code. We also want to keep the maintenance burden for developers low.
+
+We would also like to encourage researchers to use the newest action images available. We believe that providing a development environment that supports that aim will help with this.
```

Comment:
I think we should document why we'd like to encourage researchers to use the most recent action images.